src/blueprints/multilingual/templates/multilingual/editor.html

```diff
@@ -255,6 +268,30 @@ <h2 class="mb-0">
  editorPython.session.setMode("ace/mode/python");
  editorPython.setReadOnly(true);
 
+//Input Text (Prompt Message)
+  let editorExecutionInputText = ace.edit("editor-execution-input-text");
+ editorExecutionInputText.setTheme("ace/theme/dreamWeaver");
+ editorExecutionInputText.session.setMode("ace/mode/python");
+ editorExecutionInputText.setReadOnly(true);
+ editorExecutionInputText.setOption("showGutter", false);
+ editorExecutionInputText.setHighlightActiveLine(false);
+  //User Input For Execution
+ let editorExecutionInput = ace.edit("editor-execution-input");
+ editorExecutionInput.setTheme("ace/theme/dreamWeaver");
+ editorExecutionInput.session.setMode("ace/mode/python");
+ editorExecutionInput.setReadOnly(true);
+ editorExecutionInput.setOption("showGutter", false);
+ //Python Execution Output
+ let editorExecution = ace.edit("editor-execution");
+ editorExecution.setTheme("ace/theme/github");
+ editorExecution.session.setMode("ace/mode/python");
+ editorExecution.setReadOnly(true);
+ editorExecution.setOption("showGutter", false);
+ editorExecution.setHighlightActiveLine(false);
```

Comment:
All the JS concerning Skulpt should probably go in its own file (just like the convert part should also be refactored in this way).